src/ol/interaction/Draw.js

```diff
@@ -903,9 +902,44 @@ class Draw extends PointerInteraction {
   }
 
   /**
-   * Extend an existing geometry by adding additional points. This only works
-   * on features with `LineString` geometries, where the interaction will
-   * extend lines by adding points to the end of the coordinates array.
+   * Append coordinates to the end of the geometry that is currently being drawn.
+   * This can be used when drawing LineStrings or Polygons. Coordinates will
+   * either be appended to the current LineString or the outer ring of the current
+   * Polygon.
+   * @param {!LineCoordType} coordinateExtension Linear coordinates to be appended into
+   * the coordinate array.
+   * @api
+   */
+  appendCoordinates(coordinateExtension) {
```

Comment:
Minor, but naming the argument `coordinates` would make the API easier to read.

Reply:
Agreed, changed in https://github.com/openlayers/openlayers/pull/10632/commits/8d4c3b28875500484372bea736e1706b9e3dbca1